mantidproject/mslice
# Removed obsolete __future__ imports

**Description of work:**
Removed __future__ imports that are obsolete since we are not using Python 2 anymore: https://docs.python.org/3/library/__future__.html

**To test:**

Check that tests are passing. Also, the only remaining __future__ import should be the following in pyplot.py:
`from __future__ import annotations`

Fixes #1042.
